src/main/webapp/services/CommentService.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+import { getIntegrationBaseUrl } from 'symphony-integration-commons';
+import BaseService from './BaseService';
+
+const unexpectedErrorDialog = require('../templates/unexpectedErrorDialog.hbs');
+const forbiddenDialog = require('../templates/forbiddenDialog.hbs');
+const notFoundDialog = require('../templates/issueNotFoundDialog.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+const commentDialog = require('../templates/commentDialog.hbs');
+
+export default class CommentService extends BaseService {
+  constructor(serviceName) {
+    super(serviceName);
+    this.baseUrl = getIntegrationBaseUrl();
+    this.comment = '';
+  }
+
+  openCommentDialog(data, serviceName) {
+    const dialogsService = SYMPHONY.services.subscribe('dialogs');
+
+    const template = commentDialog();
+
+    const commentData = {
```

Comment:
Same as [issueStateEnricher.js](https://github.com/symphonyoss/App-Integrations-Jira/pull/87/files#diff-7ed864899feb253aa3c951c52ec56565) regarding the object creation repetition